BiconeEvolution/current_antenna_evo_build/XF_Loop/Evolutionary_Loop/Antenna_Performance_Metric/XFintoPUEO_Symmetric.py

```diff
@@ -0,0 +1,133 @@
+### File Name: XFintoPUEO.py
+### Created by: Jacob Weiler
+### Creation Date: 10/29/2022
+### Last Modified: 4/11/2023 
+### Description: This code is written to reformat the outputs of XF to be acceptable into PUEOsim. XF outputs one file, this code parses through that file and creates 6 new files. For information on inputs for PUEOsim check out this ELOG post: radiorm.physics.ohio-state.edu/elog/GENETIS/195 
+
+# General: TO RUN python XFintoPUEO.py (NPOP) (source) (Run Name) (generation) (starting individual)
+# Jacob Test: TO RUN python XFintoPUEO.py 2 . 2022_07_16_Test 0 1
+
+### THINGS TO DO: 
+### 1. Change file locations for both the inputs and outputs for this script
+### 2. Go through and change the columns and indices pulled from uan file to something that works with horn antenna XF output
+### 3. Double check to make sure outputs are correctly pulling the right gains and into the vertical and horizontal files
+
+import numpy as np
+import argparse
+import os
+import math
+from pathlib import Path
```

Comment:
Gotcha. What's the significance of the spacing between  `from pathlib import Path` and the others? Is it just because pathlib is an stdlib but we're only getting a submodule from it?